build.go

```diff
@@ -553,7 +557,13 @@ func main() {
 	if version != "" {
 		constants["main.version"] = version
 	}
-	ldflags := "-s -w " + constants.LDFlags()
+	var ldflags string
+	if isDebug {
+		// Preserve debug symbols.
+		ldflags = constants.LDFlags()
+	} else {
+		ldflags = "-s -w " + constants.LDFlags()
```

Comment:
Could you change that to only add the "-s -w" prefix if `isDebug` is false? That would avoid the code duplication